presentation/src/main/java/com/moez/QKSMS/common/QKApplication.kt

```diff
@@ -51,24 +51,36 @@ import kotlinx.coroutines.launch
 import timber.log.Timber
 import javax.inject.Inject
 
-class QKApplication : Application(), HasActivityInjector, HasBroadcastReceiverInjector, HasServiceInjector {
+class QKApplication : Application(), HasActivityInjector, HasBroadcastReceiverInjector,
+    HasServiceInjector {
 
     /**
      * Inject these so that they are forced to initialize
      */
     @Suppress("unused")
-    @Inject lateinit var analyticsManager: AnalyticsManager
-    @Suppress("unused")
-    @Inject lateinit var qkMigration: QkMigration
+    @Inject
+    lateinit var analyticsManager: AnalyticsManager
 
-    @Inject lateinit var billingManager: BillingManager
-    @Inject lateinit var dispatchingActivityInjector: DispatchingAndroidInjector<Activity>
-    @Inject lateinit var dispatchingBroadcastReceiverInjector: DispatchingAndroidInjector<BroadcastReceiver>
-    @Inject lateinit var dispatchingServiceInjector: DispatchingAndroidInjector<Service>
-    @Inject lateinit var fileLoggingTree: FileLoggingTree
-    @Inject lateinit var nightModeManager: NightModeManager
-    @Inject lateinit var realmMigration: QkRealmMigration
-    @Inject lateinit var referralManager: ReferralManager
+    @Suppress("unused")
+    @Inject
+    lateinit var qkMigration: QkMigration
+
```

Comment:
All these spacing changes will make merging incoming changes from the main repo very hard.

Reply:
It's not a big deal, we can just accept their changes in the next merge

Reply:
As I told you in Teams - I can't revert the auto formatting after I already pushed the changes, in the next merge I will accept their changes and it will be removed.